### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by AlfiYusrina**
- [x] The bug from week1 is solved.
- [x] Responsive with media query
- [x] Using fetch() instead of XHR
- [] the error message is coming up soon.

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -109,11 +109,16 @@
...
+        container.parentNode.removeChild(container);
+        const divError = document.createElement('div');
+        rootContainer.appendChild(divError);
+        divError.className = 'alert-error ';
+        divError.innerHTML = `Sorry, there's something wrong.`;
+      });
   };
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  const HYF_REPOS_URL = 'https://api.github.com/orgs1x/HackYourFuture/repos?per_page=100';
```
Comment (warrieka) [inline homework/index.js:121]: Was this, to test the error message? You should not have commited  that.  (minor error)

File: homework/index.js [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 {
   const main = url => {
+    const rootContainer = document.getElementById('root');
+    const container = document.createElement('div');
+    container.className = 'container';
+    const containerDiv = document.querySelector('.container');
```
Comment (warrieka) [inline homework/index.js:8]: you never use this containerDiv

**Comment by warrieka** [comment]
Excellent

File: homework/index.js (hunk 1 above)
Comment (AlfiYusrina) [reply homework/index.js:121]: Yes, I have fixed it on the next commit, I forgot to put it back as original

File: homework/index.js (hunk 2 above)
Comment (AlfiYusrina) [reply homework/index.js:8]: I have deleted.

**Pull request closed without merging**